config/OSS_style_idea.xml

```diff
@@ -202,7 +201,7 @@
     <option name="WRAP_ON_TYPING" value="1" />
     <indentOptions>
       <option name="INDENT_SIZE" value="2" />
-      <option name="CONTINUATION_INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
       <option name="TAB_SIZE" value="2" />
```

Comment:
Consider adding `USE_RELATIVE_INDENTS` with value true to avoid checkstyle warnings when method parameters are carried over a new line.